test/syntax/binding_when_on_syntax.test.ts

```diff
@@ -107,6 +109,7 @@ describe("BindingWhenOnSyntax", () => {
         expect(whenStub.callCount).eql(1);
         expect(whenTargetNamedStub.callCount).eql(1);
         expect(whenTargetTaggedStub.callCount).eql(1);
+        expect(whenTargetMultiTaggedStub.callCount).eql(1);
```

Comment:
Being picky but you have followed all the other poor tests that do not test that the correct arguments are passed through.